source/_patterns/03-uswds/header/secondary/_secondary-nav.twig

```diff
@@ -0,0 +1,8 @@
+<ul class="usa-nav__secondary-links">
```

Comment:
This “secondary” component is actually a sub-component of the nav component, so it should be moved there instead of having its own directory.